### ISSUE TITLE
/.htaccess wrong path

### ISSUE THREAD

**Issue by ptenchik0** [nova-framework/framework#1813]
```
    RewriteRule ^$   public/   [L]
    RewriteRule (.*) public/$1 [L]
```

now public folder a called webroot, so should be 
```
    RewriteRule ^$   webroot/   [L]
    RewriteRule (.*) webroot/$1 [L]
```

I hope more such surprises will not be

**Comment by LuckyCyborg** [comment]
You read that Nova is supposed to be installed above the site's document root, with it pointing to `webroot` folder?

When it is installed in the correct way, this `.htaccess` file is outside the area accessible to HTTP server, then have no effect.

Yet, a number of users think that they are too smartass to hear the developers recommendations and install Nova naked in the web.

So, I made this **honeypot** specially to identify those crazy bunnies. 

And, in a perfect web-server setup would produce an easy recognizable error, like:

`http://www.myshinysite.net/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public/public`

Long story short: **NOPE**. 

This is not a error, but a intentionally made trap, specially for identifying people like you, who consider that they know Nova better than me! 👎

**Comment by ptenchik0** [comment]
Hi there, thank you for your reply. I definately did't smarter than you. We all people and can forget something...
I have install framework on openserver in root dir using composer. when I try open domain where framework located and I got Internal Server Error. only after change htaccess file I was able open a website

**Comment by LuckyCyborg** [comment]
Again, the framework is not supposed to be installed this way. And is strongly recommended not to do, for security reasons and not only! Read: nasty issues comes this way.

Like I said, please install the Nova application above the site's document root, while only the files from `webroot` being visible on the web.

> We all people and can forget something...

**But** those people are supposed to read (at least) the framework's `README.md`, right? An excerpt from:

>This framework was designed and is **strongly recommended** to be installed above the document root directory, with it pointing to the `webroot` folder.
>
>Additionally, installing in a sub-directory, on a production server, will introduce severe security issues. If there is no choice still place the framework files above the document root and have only index.php and .htacess from the webroot folder in the sub folder and adjust the paths accordingly.
